### PR TITLE
feat: add configuration for automated releases

### PR DESCRIPTION
**Pull request by derberg**
Same as https://github.com/asyncapi/generator/pull/242 except of:
- no docker release
- no contribution guide and pr template as this will be handled in https://github.com/asyncapi/asyncapi/issues/355
- added step to run `npm test`
- avoided mistakes done with generator - package marked as public and removed travis and updated readme

See also: https://github.com/asyncapi/parser-js/issues/52

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
 {
-  "name": "asyncapi",
+  "name": "@asyncapi/asyncapi",
```
Comment (fmvilas) [inline package.json]: What about `@asyncapi/versions` instead?

Comment (derberg) [reply package.json]: I don't really have an opinion here, version sounds strange though, how about `spec` or `spec-schema` ?

Comment (fmvilas) [reply package.json]: `@asyncapi/specs` maybe?

**Pull request merged**